src/lambdapi.ml

```diff
@@ -121,7 +123,16 @@ let spec =
       , " Prints the current version number and exits" )
     ; ( "--debug"
       , Arg.String set_default_debug
-      , "<flags> Enables given debugging flags by default " ^ debug_flags ) ]
+      , "<flags> Enables given debugging flags by default " ^ debug_flags )
+    ; ( "--lsp-server"
+      , Arg.Unit (fun _ -> mode := LSP)
+      , " Start the LSP server" )
+    ; ( "--standard-lsp"
+      , Arg.Set Lsp.Lsp_main.std
+      , " Restrict to standard LSP protocol" )
```

Comment:
Where is this used?

Reply:
I mean: do we use a non-standard protocol @ejgallego ?

Reply:
See my other changes. This is used when we work with editors that don't support the extensions to the protocol.